### DISCUSSION TITLE
Field guard_name doesn't have a default value

### DISCUSSION THREAD

**Discussion by Ahmed-AESoliman** [spatie/laravel-permission#2232] [Q&A]
i'm trying create role and assign permission
and it's everything work good when using localhost but when  upload the app on server get error **Field guard_name doesn't have a default value**  i'm trying adding by default    like this
 `$table->string('guard_name')->default(config('auth.defaults.guard'));`
role is created role put didn't assign the permissions
and when  trying adding like this `$table->string('guard_name')->default('sanctum'); `
get error **"The given role or permission should use guard `` instead of `sanctum`."**
note:
- that's happen when upload the app on serve but in localhost work good

**Comment by angeljqv** [comment]
Did you try deleting config cache on server?
